universe/interface.go

```diff
@@ -41,14 +41,20 @@ type Identifier struct {
 	GroupKey *btcec.PublicKey
 }
 
-// String returns a string representation of the ID.
-func (i *Identifier) String() string {
+// Bytes returns a bytes representation of the ID.
+func (i *Identifier) Bytes() [32]byte {
```

Comment:
If we add this, then it should also be used in the db when we got to map the identifier into a set of bytes.

Reply:
I don't think we do that for the identifier. I can see instances where we make use of the identifier's fields. But otherwise I think we convert the identifier into a `namespace` which is the same as `Identifier.String()`. So I've added a commit to make that clear.

Reply:
Just to clarify, I was referring to this: https://github.com/lightninglabs/taproot-assets/blob/main/tapdb/universe.go#L138-L146